content/en/docs/FAQ/performance.md

```diff
@@ -3,6 +3,67 @@ title: "Performance and Scalability"
+
+#### URL Bookmarks
+
+Because most of Kiali is bookmarkable, it can be useful to save bookmarks of pages you like. Then, use these bookmarks to immediately apply the settings you want to the Kiali page you are visiting.
+
+#### Large Graphs
+
+Working with large graphs is difficult. A graph does not have to be very large before it becomes complicated and/or dense. Here are a few suggestions.
+
+##### Tips to reduce graph size and speed up generation
+
+- Limit the namespaces selected.
+  - Each requested namespace is like it's own graph request, and then each resulting namespace graph is "stitched" together.
+  - This may not be possible, in some mesh designs even a single namespace is very populated.
+- Reduce the protocols selected
+  - Using the Traffic dropdown, only fetch TCP or HTTP, not both. Different queries are performed for the different protocols.
+  - In Ambient, you can also choose between ztunnel and waypoint telemetry. This can reduce the number of queries and/or the size of your graph.
+- Prefer smaller Duration dropdown values.
+  - The larger the duration the more metric data being processed.
+- Enable response time edge labels only after minimizing the size of your graph.
+  - This requires extra queries against Prometheus histograms, and can be expensive.
+- Enable the Security Display option only after minimizing the size of your graph.
+  - This requires extra Prometheus queries.
+- Disable the Service Nodes Display option, if not needed.
+  - This is enabled by default, and provides valuable routing information, but it does also add extra nodes and edges.
+- Only enable Operation Nodes as needed.
+  - This option is very valuable when using request classification, but does require extra queries, and does add extra nodes and edges.
+
+##### Tips for manipulating your graph
+
+After your graph is generated and rendered in the UI, there are client-side ways to improve your visualization:
+
+- Graph Find and Hide
+  - Find and Hide are very valuable tools. Both use the same simple query language, described in detail in the on-screen help (click the info icon next to the inputs, on the toolbar). It is highly recommended to become familiar with this feature, very simple expressions can be useful.
+  - Find will highlight the nodes and edges that match the expression. This can help locate nodes and edges in a large graph (or even a small graph).
+  - Hide will temporarily remove the matching nodes and edges. This can effectively clean up a large graph into a very focused view.
+  - It is possible to pre-define Find and Hide expressions in your Kiali CR. These pre-defined expressions can even be configured to be applied automatically.
```

Comment:
If I were reading this, I would say, "Oh, yes. I like that idea. How do I set up and automatically use pre-defined expressions?" Do we have other docs you can put a link to here to help the user go find out how to do this? Something like: `For more information on this feature see <link here>`

Reply:
OK, I'll add more detail.

Reply:
See the new sub-bullet